runtime.js

```diff
@@ -133,6 +133,22 @@ export async function hide (index = window.__args.index) {
   return await ipc.send('hide', { index })
 }
 
+/**
+ * @param {object} opts - an options object
+ * @param {number} [opts.window = window.__args.index] - the index of the window
+ * @param {number} opts.value - the path to the HTML file to load into the window
+ * @return {Promise<ipc.Result>}
+ */
+export async function navigate ({ window: _window, value: _value }) {
+  const index = _window ?? window.__args.index
+  const value = `file://${window.__args.cwd()}/${_value}`
+  return await ipc.send('navigate', { index, value })
+}
+
+export async function openProcess () {
+  return await ipc.send('process.open')
+}
```

Comment:
I think we need to move this to a separate module. This way tree shaking will work better. WDYT about `backendProcess.js` naming? @heapwolf @jwerle
Related issue https://github.com/socketsupply/socket/issues/23

Reply:
or maybe just `backend`, so methods will be `io.backend.open({ force: boolean })`, `io.backend.kill()`, `io.backend.isOpen()`

Reply:
moved to `backend.js`